docs/integrations/airflow.md

```diff
@@ -0,0 +1,128 @@
+# Airflow
+
+Airflow has a native OpenTelemetry integration for [traces] and [metrics].
+
+Airflow creates an exporter internally, which sends data to the configured backend.
+
+Unfortunately, it's not possible to send data to Logfire directly, yet... But we are working on it. :wink:
+
+??? question "Why it's not possible to send data directly?"
+    The Airflow implementation doesn't allow to override the headers used by the exporter.
+    See [apache/airflow#44346](https://github.com/apache/airflow/pull/44346) for more details.
+
+    We need the ability to set the authorization header to send data to **Logfire**.
+    See our [Alternative clients section](../guides/advanced/alternative-clients.md).
+
+## Setup
+
+To enable the OpenTelemetry integration in Airflow, you'll need to have an [OpenTelemetry Collector] running, and then
+configure Airflow to send data to it.
+
+### OpenTelemetry Collector
+
+The OpenTelemetry Collector is a vendor-agnostic agent that can collect traces and metrics from your applications and
+send them to various backends.
+
+In this case, we are interested in sending data to **Logfire**. :fire:
```

Comment:
This feels like too many words to say you should send data from airflow to a collector which in turn sends it to logfire. And it starts with "Unfortunately, it's not possible to send data to Logfire directly" and takes too long to make it clear that you can send it indirectly.

Reply:
I'll leave as is, and when @adriangb works on the OTel Collector section, we can remove this whole thing.